### ISSUE TITLE
State: Finalize Application screens

### ISSUE THREAD

**Issue by mchung07** [openopps/OpenOppsTasks#102]
https://opm.invisionapp.com/d/main/#/projects/prototypes/15360465

Need to review the screens with State and get their final feedback

**Comment by humancompanion** [comment]
@kbwagner and @tlorrison - Min is waiting on final review and feedback. When can we expect that? Min needs to roll off of State work onto the North Star project a.s.a.p. Thanks.

**Comment by kbwagner** [comment]
@humancompanion sorry didn't know she was waiting on this from me.  I think this is in need of a meeting with State.  @tlorrison can we set something up with them?

**Comment by mchung07** [comment]
We still need to discuss what we'll do with transcript
Related issue: https://github.com/openopps/OpenOppsTasks/issues/114

**Comment by kbwagner** [comment]
Will review with State 11/15/18.  Outstanding questions for application include:
1) Transcripts - separate task 124 and 125
2) Eligibility screen out - separate task 128.
3) Footer - should we include their email address for help instead of ours?
4) PRA - do we need it?  If they have one do we need to post their number somewhere?
